website/content/docs/operations/aws-oidc-provider.mdx

```diff
@@ -0,0 +1,403 @@
+- [`<CERT_ARN>`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_listener#certificate_arn):
+The AWS resource name of the load balancer's certificate. This will be
+`aws_acm_certificate_validation.example.certificate_arn` if you are using the examples from this
+page.
+- [`<LB_TARGET_GROUP_ARN>`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_listener#target_group_arn): The AWS resource name of the target group for the load balancer to route traffic to. This group
+includes your Nomad server instances.
+
+### Create a DNS Alias
+
+Use the [`aws_route53_record` resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record).
+
+```hcl
+resource "aws_route53_record" "www" {
+  zone_id = <HOSTED_ZONE_ID>
+  name    = var.domain_name
+  type    = "A"
+
+  alias {
+    name                   = <LB_ALIAS_DNS_NAME>
+    zone_id                = <LB_ALIAS_ZONE_ID>
+    evaluate_target_health = true
+  }
+}
+```
+
+This configuration requires the following information:
+- [`<HOSTED_ZONE_ID>`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record#zone_id):
+The ID of the hosted zone. This will be `aws_route53_zone.example.zone_id` if you are using the
+examples from this page.
+- [`<LB_ALIAS_DNS_NAME>`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record#name):
+The DNS
+name of the load balancer. This will be `aws_lb.test.dns_name` if you are using the examples
+from this page.
+- [`<LB_ALIAS_ZONE_ID>`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record#zone_id):
+The zone ID of the load balancer. This will be `aws_lb.test.zone_id` if you are using the
+examples from this page.
+
+### Create an OIDC Identity Provider
+
+Use the [`aws_iam_openid_connect_provider` resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_openid_connect_provider).
+
+```hcl
+data "tls_certificate" "example" {
+  url = <CERT_DOMAIN_NAME>
+}
+
+resource "aws_iam_openid_connect_provider" "nomad" {
+  # Nomad HTTPS URL
+  url = <CERT_DOMAIN_NAME>
+
+  client_id_list = [
+    "aws",
+  ]
+
+  thumbprint_list = [data.tls_certificate.example.certificates.0.sha1_fingerprint]
+}
+```
+
+This configuration requires the following information:
+- [`<CERT_DOMAIN_NAME>`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_openid_connect_provider#url):
+The domain name of the load balancer certificate. This will be
+`aws_acm_certificate.example.domain_name` if you are using the examples from this page.
+
+### Create an IAM policy for OIDC Federated Users
+
+Use the [`aws_iam_role` resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role)
+to create an appropriate IAM role for federated users. This will be specific to your use case.
+The following example sets S3 bucket access.
```

Comment:
Although the AWS docs use "users", we're always talking about workloads here, so clarifying that might be helpful:

```suggestion
to create an appropriate IAM role for workloads acting as federated users. This will be 
specific to your use case. The following example allows workloads access to S3 buckets.
```